quantized_mesh_encoder/occlusion.py

```diff
@@ -30,8 +30,8 @@ def compute_magnitude(positions, bounding_center):
 
 
 # https://cesiumjs.org/2013/05/09/Computing-the-horizon-occlusion-point/
-def occlusion_point(positions, bounding_center):
-    ellipsoid = np.array([RADIUS_X, RADIUS_Y, RADIUS_Z])
+def occlusion_point(positions, bounding_center, ellipsoid = WGS84):
+    ellipsoid = np.array([ellipsoid["a"], ellipsoid["a"], ellipsoid["b"]])
```

Comment:
This seems like a place where we'd want to allow the ellipsoid to have three axes?